examples/DataApi-Voting-poll/template/modules/data_l1/src/main/scala/com/my/currency/data_l1/Main.scala

```diff
@@ -2,20 +2,23 @@ package com.my.currency.data_l1
 
 import cats.data.NonEmptyList
 import cats.effect.IO
-import cats.implicits.catsSyntaxValidatedIdBinCompat0
-import com.my.currency.data_l1.CustomRoutes.{getAllPolls, getPollById}
-import com.my.currency.shared_data.MainData
-import com.my.currency.shared_data.MainData.{PollUpdate, State}
+import cats.implicits.{catsSyntaxApplicativeId, catsSyntaxValidatedIdBinCompat0}
```

Comment:
```suggestion
import cats.syntax.applicative._
import cats.syntax.validated._
```

Reply:
Done

Reply:
@sdavidp about the point of getCalculatedState, I think we may want to return a 400 or 500 and with some message that state failed to be obtained

Reply:
Got it, fixed